src/main/java/org/jenkinsci/test/acceptance/po/WizardCustomizeJenkins.java

```diff
@@ -0,0 +1,74 @@
+package org.jenkinsci.test.acceptance.po;
+
+import static org.jenkinsci.test.acceptance.Matchers.hasContent;
+
+import java.util.concurrent.Callable;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.NoSuchFrameException;
+import org.openqa.selenium.WebElement;
+
+/**
+ * Page object for Wizard Customize Jenkins Page.
+ *
+ */
+public class WizardCustomizeJenkins extends PageObject {
+
+    public WizardCustomizeJenkins(Jenkins jenkins) {
+        super(jenkins.injector, jenkins.url(""));
+    }
+
+    public WizardCustomizeJenkins doInstallSuggested() {
+        By suggestedButtonLocator = By.partialLinkText("Install suggested plugins");
```

Comment:
I would suggest using `by` if possible, `by` is an instance of [ByFactory](https://github.com/jenkinsci/acceptance-test-harness/blob/69b1fae7987c219022d999f45e5dee44e8a69524/src/main/java/org/jenkinsci/test/acceptance/ByFactory.java), so if you need a new selector and simply use it via Selenium in your PO the new selector can not be reused, but if you add it to `ByFactory` it is available on all PO, also you can add some validation or error control logic easily without polluting your PO